adage/adage/settings.py

```diff
@@ -23,10 +24,17 @@
     CONFIG = CI_CONFIG
     pg_user = os.environ.get('PG_USER')
     pg_pass = os.environ.get('PG_PASSWORD')
-    CONFIG['dbmaster']['USER'] = pg_user
-    CONFIG['dbmaster']['PASSWORD'] = pg_pass
     CONFIG['databases']['default']['USER'] = pg_user
     CONFIG['databases']['default']['PASSWORD'] = pg_pass
+elif os.environ.get('CIRCLECI') == 'true':
+    with open(os.path.join(BASE_DIR, 'adage', 'config.py.template')) as f:
+        exec f
+    CONFIG = CI_CONFIG
+    CONFIG['databases']['default']['NAME'] = 'circleci_test'
+    CONFIG['databases']['default']['USER'] = 'ubuntu'
+    CONFIG['databases']['default']['PASSWORD'] = ''
```

Comment:
Replying to the e-mail didn't produce this comment in precisely the right place. So I'll put it here and delete the email response (sorry for the duplicate e-mails):
> That is how circleci configures posters in their testing environment.